packages/amplication-client/src/Components/SvgThemeImage.scss

```diff
@@ -1,8 +1,8 @@
 @import "../style/index.scss";
 
 .svg-theme-image {
-  width: 150px;
-  height: 150px;
+  width: 250px;
+  height: 250px;
```

Comment:
Changing the default size of "SvgThemeImage" will impact other places and not only the app list page

Please do one of the following:

1. Add property to the SvgThemeImage component for size (something like small/medium/large) and set the size in CSS based on the size property using a new class

2. override the size of the image in the ApplicationList.scss 

.application-list {
  &__empty-state {
     .svg-theme-image {
        width: 250px;
         height: 250px;
     }
   }
}

Reply:
okay, doing the changes.